docs/data_engineers/accessing_data/elt_pipeline/elt_destinations/snowflake.md

```diff
@@ -0,0 +1,301 @@
+---
+sidebar_position: 2
+---
+
+# Snowflake Destination
+
+In order to create an end to end test of the ELT pipeline apart from the Source which is the KYVE network, you should
+configure a destination where the `records` will be stored.
+
+This page will guide you through setting up an ELT pipeline using Snowflake to fetch data from
+a [KYVE data pool](https://docs.kyve.network/basics/pools.html) and import it into a `Snowflake` database.
```

Comment:
nit but I think we should refer to Snowflake as a "data warehouse" rather than database